### PR TITLE
Fixing note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/2137

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/let-users-connect-classic-sites-to-groups.md [hunk 1]
```diff
@@ -30,7 +30,6 @@ As a global or SharePoint admin in Microsoft 365, you can allow or prevent site
 1. Go to the [Settings page of the new SharePoint admin center](https://admin.microsoft.com/sharepoint?page=settings&modern=true), and sign in with an account that has [admin permissions](/sharepoint/sharepoint-admin-role) for your organization.
 
 >[!NOTE]
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/let-users-connect-classic-sites-to-groups.md:32]:
```suggestion
> [!NOTE]
```

File: SharePoint/SharePointOnline/let-users-connect-classic-sites-to-groups.md [hunk 2]
```diff
@@ -30,7 +30,6 @@ As a global or SharePoint admin in Microsoft 365, you can allow or prevent site
...
-<<<<<<< Updated upstream
 >If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center and open the Settings page. <br>If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center and open the **Settings** page.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/let-users-connect-classic-sites-to-groups.md:33]:
```suggestion
> If you have Office 365 Germany, [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=848041), then browse to the SharePoint admin center and open the Settings page. <br>If you have Office 365 operated by 21Vianet (China), [sign in to the Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=850627), then browse to the SharePoint admin center and open the **Settings** page.
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**